app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [Person(name=human["name"], age=human["age"])
+               for human in people]
+    for human in people:
+        if "wife" in human and human["wife"]:
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif "husband" in human and human["husband"]:
+            Person.people[human["name"]].husband\
```

Comment:
I'd prefer to create sume variables to contain a long-named data units like "Person.people[human["name"]].wife etc. Flake8 app is not complaining about that, as I see, but it's a good thing for making your code easy to read.